cmd/gitbase/server.go

```diff
@@ -85,5 +95,7 @@ func (c *CmdServer) Execute(args []string) error {
 		return err
 	}
 
+	logrus.Debug("starting server")
```

Comment:
maybe we should add a defer logrus.Debug("server started")

Reply:
Start never returns until the connection stops, so a defer would actually be "stopping server" 😛

Reply:
oh you're right